packages/lesswrong/components/posts/PostsTitle.tsx

```diff
@@ -37,8 +38,16 @@ const styles = (theme: ThemeType): JssStyles => ({
     paddingRight: 10,
     position: "relative",
     top: 2,
-    color: theme.palette.icon.slightlyDim3,
+    color: theme.palette.icon[isEAForum ? "dim4" : "slightlyDim3"],
   },
+  stickyIcon: isEAForum
+    ? {
+      width: 16,
+      height: 16,
+    }
+    : {
+      fontSize: "1.3rem",
```

Comment:
This feels slightly too large compared to the current LW site, I would prob go with 1.2rem